tsconfig.json

```diff
@@ -15,5 +15,6 @@
     },
     "allowSyntheticDefaultImports": true
   },
+  "files": ["src/server/types.d.ts"],
```

Comment:
Move types to `/types` folder. Should be 1 file per type. I.e. here you'll have `/src/types/express.d.ts`